packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -727,11 +1458,30 @@ export class ApexDebug extends DebugSession {
   }
 
   private handleStopped(message: DebuggerMessage): void {
+    const reason = message.sobject.BreakpointId ? 'breakpoint' : 'step';
     const threadId = this.getThreadIdFromRequestId(message.sobject.RequestId);
+
+    this.log(
+      TRACE_CATEGORY_LAUNCH,
+      `handleStopped: got ${reason} event from server for thread ${threadId}`
```

Comment:
This doesn't seem to fit the launch category.

Reply:
Would you think that "lifecycle" is a better term than "launch"? The launch category is used for general debugger events - launched/started/connected/stopped

Reply:
I think a category specifically for apex debugger events would fit better.